### ISSUE TITLE
ContextAwareDataPersisterInterface & symfony/framework-bundle 5.3.7

### ISSUE THREAD

**Issue by nayluge** [api-platform/core#4438]
**API Platform version(s) affected**: 2.6.5

**Description**  
I'm upgrading to the last version of the framework and get this error on composer install:

!!    Cannot autowire service "App\DataPersister\ProductDataPersister": argument   
!!    "$decorated" of method "__construct()" references interface "ApiPlatform\Co  
!!    re\DataPersister\ContextAwareDataPersisterInterface" but no such service ex  
!!    ists. Try changing the type-hint to "ApiPlatform\Core\DataPersister\DataPer  
!!    sisterInterface" instead.   

Works like a charm with the previous framework bundle version.

**How to reproduce**  
Create a custom decorated DataPersister like defined on the doc : 
https://api-platform.com/docs/core/data-persisters/#decorating-the-built-in-data-persisters

**Possible Solution**  

**Additional Context**

**Comment by WhiteRabbitDE** [comment]
Can you share your services.yaml ? I had the same error today, copied from an older project where it was working. 
My error was solved after explicitly adding [.inner] as Argument. I didnt need it in my older project somehow

**Comment by nayluge** [comment]
It's actually autowired : 

```
    App\DataPersister\ProductDataPersister:
        decorates: 'api_platform.doctrine.orm.data_persister.inner'
```

I'll try to define it manually

**Comment by WhiteRabbitDE** [comment]
That ist how i had it before, and then i had to add the argument manually. At least for the Data Persister. Not for the other Arguments.

**Comment by nayluge** [comment]
No change with definition of the argument.

It only works if i alias it this way : 

```
ApiPlatform\Core\DataPersister\ContextAwareDataPersisterInterface: '@ApiPlatform\Core\DataPersister\DataPersisterInterface'

```

**Comment by hosseinsalemi** [comment]
Remove all previous data persister config from `services.yaml` and add below code to `services.yaml`

```
App\DataPersister\:
    resource: '../src/DataPersister'
    arguments: ['@api_platform.doctrine.orm.data_persister']
```

**Comment by Chris53897** [comment]
@nayluge i guess it is this error https://github.com/symfony/symfony/pull/42815 fixed in 5.3.8

Can this ticket get closed?
